tests/internal/e2e/test/specs/vcast_coded_tests.test.ts

```diff
@@ -27,7 +27,7 @@ describe("vTypeCheck VS Code Extension", () => {
   let bottomBar: BottomBarPanel;
   let workbench: Workbench;
   let statusBar: StatusBar;
-  const TIMEOUT = 120_000;
```

Comment:
Add a comment please about why we need this.
I realize you only bumped it from 2 to 3 but check with Igor about why its there anyway

Reply:
This timeout increase is primarily for my setup, as my VM is somewhat slow during E2E execution. Otherwise, the tests fail locally.